src/lib/components/ContentfulRichText/nodes/EntryHyperlink.svelte

```diff
@@ -16,13 +16,14 @@
 
   const linksContext = getContext<LinksContext | undefined>(linksKey);
   if (!linksContext) throw new Error("no context was provided for entry hyperlink");
+
   const { id: entryID } = entryHyperlink.data.target.sys;
 
   const entry = linksContext.linksEntriesMaps.hyperlink.get(entryID);
 
   let entryMetadata: PageMetadataMapItem;
```

Comment:
This is one way to clear up the type error
```suggestion
  let entryMetadata: PageMetadataMapItem | undefined;
```